pnpm-lock.yaml

```diff
@@ -1055,8 +1055,8 @@ packages:
       '@warp-ds/uno': 1.3.0
     dev: true
 
-  /@warp-ds/icons@1.1.0:
-    resolution: {integrity: sha512-2EJhU6L2nhaM3iTC9uApe5KU5/Pb5oNC6UncClfo5VMSjBzUn4RNkwyiyJlk4FrL41P7kre30M+VXpr+3KclFA==}
+  /@warp-ds/icons@1.2.0:
```

Comment:
Oh, we didn't update icons to 1.3.0 in vue, but I guess that won't be a problem?

Reply:
Maybe in the next one then 👍